### ISSUE TITLE
Remove exception phrasing from 1.2.1, 1.2.2, and 1.2.3

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#796]
There is inconsistent phrasing used within the SC of Guideline 1.2.  This should be corrected.

SC 1.2.1, 1.2.2, and 1.2.3 all include the condition:  "except when the media is a media alternative for text and is clearly labeled as such".

This conditional is _not_ used in other SC with similar structure (namely 1.2.5, 1.2.6, 1.2.7, and 1.2.8).

The conditional is not necessary (in the SC) because the [normative definition for synchronized media](http://www.w3.org/TR/WCAG20/#synchronizedmediadef) already includes that qualification.
> audio or video synchronized with anotherformat for presenting information and/or with time-based interactive components, **unless the media is a media alternative for text that is clearly labeled as such** 

In other words, media alternatives for text are _not_ synchronized media.

Proposal:  In 2.2, we should remove the exception phrasing from 1.2.1, 1.2.2, and 1.2.3.

**Comment by alastc** [comment]
Hi Bruce,

That sounds logical (and thank you for tagging the issue appropriately). I do wonder though if it is right to rely on a definition to provide an important exception?

What doesn't seem logical (to me) to say that: This element on the page (e.g. a video with audio) **_is not_** syncronised media based on something else on the page.

I'd rather remove the exception from the defintion and add it to the SCs, unless there is some process reason that doesn't work.

**Comment by bruce-usab** [comment]
I agree that removing the exception from definition (and adding it to the SCs) is also a possibility.

I also agree that having an exception in definition is weird.

OTOH, the SC without the explicit exceptions are so much easier to read!

But I think that the readability is trumped by your observation about how illogical it is to have certain video with audio **not** be synchronized media _based on something else on the page_.

So, I agree with your fix.

(sorry i tried, but could not get the tag widgets to trigger correctly with the issue i created before this one.  i got it figured this time around, obviously)

**Comment by alastc** [comment]
Hi Bruce,

I created a PR to make this update (pending approval) in #806.

When I got to "Sign Language (Prerecorded)" at AAA, I did wonder if we really want that condition at AAA?

Also, I tend to consider sign-language a translation, and even if there is equivelent text on the page it wouldn't be equivelent for someone who needs sign language. In which case, don't have the condition/exception at AAA?

What do you think? 

If we don't apply the condition it's probably best done as a WCAG 2.2 thing as the requirements are tightening up. Otherwise just moving the condition from the definition to the SCs could be an errata (I think).

**Comment by mbgower** [comment]
I definitely think the exception should be called out in the understanding document, if it removed from the normative text.

**Comment by bruce-usab** [comment]
Per call 7/14, i will review PR, especially as it impacts AAA SC.

**Comment by bruce-usab** [comment]
We discussed on call today, 4/13.

1.  I don't think ever took a pass at reviewing against 1.2.8.
2. As I understood the next steps, @alastc offered to write up an analysis, which will inform the exact edits required.
3. The consensus on the call was that, if there is a recommendation for an edit (quite probable), the PR should be formatted as an  update to errata.  Here are links to [WCAG 2.0 errata](https://www.w3.org/WAI/WCAG20/errata/) and [2.1 errata](https://www.w3.org/WAI/WCAG21/errata/).

**Comment by awkawk** [comment]
Here's my breakdown of the SC in 1.2:

SC # | SC Name | Level | Calls out Media Alternative for   text exception | References Synchronized Media | Full Text Transcript | Caption track | Audio Description track | Sign Language Track
-- | -- | -- | -- | -- | -- | -- | -- | --
1.2.1 | Audio-Only and Video-only - AUDIO PART | A | Y | N | x |   |   |  
1.2.1 | Audio-Only and Video-only - VIDEO PART | A | Y | N | x |   | x |  
1.2.2 | Captions (Prerecorded) | A | Y | Y |   | x |   |  
1.2.3 | Audio Description or Media Alternative (Prerecorded) | A | Y | Y |   |   |   |  
1.2.4 | Captions (Live) | AA |   | Y |   | x |   |  
1.2.5 | Audio Description (Prerecorded) | AA |   | Y | x |   | x |  
1.2.6 | Sign Language (Prerecorded) | AAA |   | Y |   |   |   | x
1.2.7 | Extended Audio Description (Prerecorded) | AAA |   | Y |   |   | x |  
1.2.8 | Media Alternative (Prerecorded) | AAA |   | Y | x |   |   |  
1.2.9 | Audio-only (Live) | AAA |   | N | x |   |   |  

What I concluded is that the only SC that needs to include the "except when the audio or video is a media alternative for text and is clearly labeled as such" phrase is 1.2.1 because it don't reference the synchronized media term (this SC is on two rows because there are two parts to it). The only two that reference the term and also include the phrase are 1.2.2 and 1.2.3. 

So, the problem is that people reading 1.2.2 and 1.2.3 may be confused since both the SC and the definition include the phrase, and that may make people think that 1.2.4-.8 are different in this way when they are not.

If we change the definition, we will need to change 1 definition and 5 SC.

If we change the mention of the phrase we can just remove it from two SC.

I don't see a backward compatibility issue and can see that the intent of the WG in 2008 was to provide the exception in 1.2.2 and 1.2.3 - the only problem is that it is mentioned twice. I could support removing the phrase from these SC as an errata, but do wonder if it is worth it.

**Comment by alastc** [comment]
My main point is that if you are trying to decide whether you need to add audio description at level AA, you read down the SCs:

- 1.2.1 video only: Calls out the media alternative exception.
- 1.2.3: Calls out the media alternative exception.
- 1.2.5: Does not call out the exception.

How about we just add the exception text to 1.2.5 and leave the definition alone? 
That at least would make 1.2.3 and 1.2.5 consistent.

The effect of this omission is that some accessibility companies, members of our group, and people who have read all the documentation do not realise that the media-alternative is an allowed approach at AA.

**Comment by awkawk** [comment]
@alastc 

> The effect of this omission is that some accessibility companies, members of our group, and people who have read all the documentation do not realise that the media-alternative is an allowed approach at AA.

The only time that the media alternative would be ok for 1.2.5 is if the video has no real content (e.g., talking head video) so that there is nothing to describe. This is a significant judgement call for a describer to make and requires a bit of gymnastics in reading the SC text and definitions. To arrive at that I think that you basically need to think "we have described it in adjacent text because there is so little to describe that synchronization with the audio content isn't relevant" or you need to regard the visual part of the video as an alternative for the text that exists adjacent to it, which is not only difficult to argue with a straight face but I believe I have yet to see a video on the internet in my life that clearly indicates that the video portion is provided as an alternative to text except when the video is in sign language.

I think that the Understanding document is pretty clear on this:

> At Level A in Success Criterion 1.2.3, authors do have the choice of providing either an audio description or a full text alternative. If they wish to conform at Level AA, under Success Criterion 1.2.5 authors must provide an audio description - a requirement already met if they chose that alternative for 1.2.3, otherwise an additional requirement. 

So, I don't think that just adding the phrase to 1.2.5 makes sense. If we felt like it should be added to the SC we would need to be consistent and add it to all five that are missing it otherwise we would be soon receiving requests to do so saying that we just shifted the confusion to a different SC.

**Comment by mraccess77** [comment]
The challenge is that media alternative is used two different ways
* in 1.2.3 it's used as a way to provide an alternative instead of audio description to meet level A.  
* In all other SC it is used as a way to say that you don't need to make some video only, audio only, or combination of video and audio content accessible because it is a media alternative to the text.   So someone could say that a synchronized video does not need captions or audio descriptions because the video itself is secondary to the text on the page.  This does mean the text would have to meet a high bar and be labeled as the media alternative - but ultimately it could be used as a loop hole to not have to provide captions or audio description.  Likely the text on the page would need to be considered the primary content and the video provided primarily as secondary content.

**Comment by bruce-usab** [comment]
Thanks @alastc for that additional information on the errata mechanics.

> If we change the mention of the phrase we can just remove it from _two_ SC.

_Three_ SC, no?  1.2.1, 1.2.2, 1.2.3

@awkawk I agree that Understanding document is pretty clear.  That does not seem to be enough.  If 2.0 is roughly drafted so as to trip up @alastc, that is compelling evidence to me that we need an errata!

The problem I perceive with 1.2.5 is when an ASL video is provided as a media alternative for (a static) text document.  (This is rare.)  An audit could come across that and reasonably ask:  <q>Where is the audio description?</q>  (It the auditor asks, <q>Where are the captions?</?> [per 1.2.2], the exception is part of the SC.

It is the non-parallel phrasing which leaves room for confusion.

**Comment by awkawk** [comment]
> Three SC, no? 1.2.1, 1.2.2, 1.2.3 ?

No, because 1.2.1 doesn't reference the Synchronized media term, so that phrase is needed on 1.2.1.

> The problem I perceive with 1.2.5 is when an ASL video is provided as a media alternative for (a static) text document. (This is rare.) An audit could come across that video and reasonably ask: Where are the 1.2.2 captions and the 1.2.5 audio description? For 1.2.2, the exception is part of the SC. For 1.2.5, one has to look at Understanding (and the definition).

The sign language video wouldn't be evaluated under 1.2.5 generally as it is video-alone, but all people need to do is read the definition. 

@mraccess77 I think that you are confusing "alternative for time-based media" and "media alternative for text".

**Comment by mraccess77** [comment]
@awkawk I agree that an alternative to time-based media is different - but the confusion is that the short bold name of the SC 1.2.3 says "media alternative" "1.2.3 Audio Description or Media Alternative (Prerecorded): ".  I think it should read "1.2.3 Audio Description or Alternative for Time-based Media (Prerecorded): "

As for media alternatives - some interpretation appears to say that a media alternative would only be allowed under SC 1.2.5 when the media alternative is video only - whereas I don't see anything that puts any restrictions on it.  That is for SC 1.2.2 and 1.2.5 if we had a synchronized video with audio and video someone could claim that video is a media alternative to the text on the page and would not be required to provide captions or audio descriptions.

**Comment by bruce-usab** [comment]
@awkawk agreed, the ASL video would not need audio, so would fall under 1.2.1.

There have also been COGA folks asking for videos (featuring humans) as an alternative for text documents.  Accessibility advocates would want those (closed) captioned (since the dual modality of information helps some COGA folks), but 1.2.2 exempts the captioning requirement.  On the other hand, 1.2.5 requires audio description, regardless that any extraneous audio would decreases the utility for _all_ audience members.

**Comment by awkawk** [comment]
@bruce-usab 

> There have also been COGA folks asking for videos (featuring humans) as an alternative for text documents. Accessibility advocates would want those (closed) captioned (since the dual modality of information helps some COGA folks), but 1.2.2 exempts the captioning requirement. On the other hand, 1.2.5 requires audio description, regardless that any extraneous audio would decreases the utility for all audience members.

People might decide to provide CC for a video that is an alternative to a text document, but WCAG doesn't require that. Nor would that video need to be audio described because it is a media alternative to text (assuming that it is marked appropriately). The definition of synchronized media includes this exception for 1.2.5 and for 1.2.2, even though 1.2.2 restates that exemption in the SC text.

> I am still not quite groking why 1.2.1 is so different than 1.2.2 and 1.2.3, but let me think about that!
1.2.1 is audio only or video only, so the media is not synchronized. 1.2.2 and 1.2.3 are talking about synchronized media (audio and video together) and specifies how to make the audio in synchronized media and the video in synchronized media more accessible. For 1.2.1 the exemption for media alternatives for text is still appropriate but it needs to be mentioned explicitly since there is no reference to synchronized media where that phrase is found.

@mraccess77 yes, I agree that would be a better title and would avoid some confusion. Regarding people claiming that a video is an alternative to text and not making the video accessible - yes, they could do that. If I had a video of the President making remarks at some event and the same page included a text transcript of the remarks and context, as long as people know that these are the same content and one is not supplementary to the other, that would be ok under WCAG 2.x. It would still be better to include closed captioning for the text as spoken, and it would be better to include audio description (although the AD might just be "President Biden speaks from a podium in the East room of the Whitehouse"), but not required. Agree?

**Comment by alastc** [comment]
AWK wrote:
> The only time that the media alternative would be ok for 1.2.5 is if the video has no real content (e.g., talking head video) so that there is nothing to describe.

I can't square that with the definition then, if you combine 1.2.5 with it's definition you get:
"Audio description is provided for all prerecorded video content in audio or video synchronized with another format for presenting information and/or with time-based interactive components, **unless the media is a media alternative for text that is clearly labeled as such**"

I thought you had agreed with that interpretation in the [thread on-list](https://lists.w3.org/Archives/Public/w3c-wai-gl/2020JulSep/0189.html).

I've had a few scenarios with clients where it is either a:
- medical condition page
- support page (how to do X with your product)
- product page

And the page as a whole includes all the information (and more) in text, and the video is another way of conveying the same thing. In that scenario, with the correct labelling of the video, doesn't that pass?

> So, I don't think that just adding the phrase to 1.2.5 makes sense.

In how we use definitions, it is already there, it's just hidden.

Bruce wrote:
> the caveat — "and is clearly labelled as such" — is not part of the definition

It's in the synchronised media definition, rather than the media-alt for text definition. That's why it isn't part of 1.2.1 which isn't for synchronised media.

**Comment by awkawk** [comment]
> I can't square that with the definition then, if you combine 1.2.5 with it's definition you get: "Audio description is provided for all prerecorded video content in audio or video synchronized with another format for presenting information and/or with time-based interactive components, unless the media is a media alternative for text that is clearly labeled as such"

I read it as "audio description is provided for video content that is synchronized with another format unless that media (the video that is synchronized with the other format) is a media alternative for text and is labeled as such". Given that there is usually some content in the video that merits description this is an extreme edge case which also requires that the video is clearly identified as a media alternative to text. 

I think that what I wrote in that thread is consistent with what I'm saying. Maybe you're hearing something else, I'm not sure what specifically.

> And the page as a whole includes all the information (and more) in text, and the video is another way of conveying the same thing. In that scenario, with the correct labelling of the video, doesn't that pass?

Yes.

> In how we use definitions, it is already there, it's just hidden.

Yes. But why would we "correct" the hidden nature for 1.2.5 and not the other SC? Easiest is to just remove the duplicate phrase from 1.2.2 and 1.2.3.

**Comment by mraccess77** [comment]
@awkawk wrote "unless that media (the video that is synchronized with the other format) is a media alternative for text and is labeled as such". 

That's not how I read the media alternative exception - I read it has applicable to any synchronized content that acts a media alternative to the page content.  Your narrower interpretation may be preferred but that's not how I read the description of synchronized media.  So clearly we all need to come to some consensus on that first before deciding on a solution.

**Comment by awkawk** [comment]
@mraccess77 I'm not getting what is different in your view...

**Comment by alastc** [comment]
> Given that there is usually some content in the video that merits description this is an extreme edge case

I think this might be the divergence. 

If you think about it as: Is there content in the video that could be described? That's a high bar.

If you think about it as: Can you gather the same (or same + more) information from the page content as you would from an audio description? That isn't such a high bar.

There is quite a big difference between what _could_ be described, and what is _useful_ to describe.

Given the definition of "media alternative for text", any of the scenarios above could fit that.

Bruce wrote:
> I agree that Understanding document is pretty clear.

I don't agree there, the understanding doc is very short and doesn't mention "media alternative for text" outside of the definition. The note in the intent comes across as about audio desc and transcripts, NOT media alternatives. It is silent on that aspect.

AWK wrote:
> why would we "correct" the hidden nature for 1.2.5 and not the other SC? Easiest is to just remove the duplicate phrase from 1.2.2 and 1.2.3.

Putting an exception in the definition increases confusion, it hides it. The only other example of that I can find is in the general flash threshold, and even there it is an exception for the definition of flashing, rather than the context of the page.

I see the point about consistency, I've been focused on the audio-described aspect. 

So my preference would be to add the exception to the SC text for any applicable SC. It is adding duplication, but also clarity.

**Comment by Ryladog** [comment]
I think this is an excellent agenda item - BUT - after-hours in-person
face-to-face meeting topic - over adult beverages.

Somehow, we have to agree on what is best, what is meant, and then provide
a simpler explanation in an eratta of WCAG 2s and for WCAG 3.

**Comment by mraccess77** [comment]
@awkawk you wrote "Given that there is usually some content in the video that merits description this is an extreme edge case".  

I couldn't tell what you meant by that.  I now believe you are saying that most pages wouldn't contain the same amount of detailed text and thus most situations could not use the media alternative approach because the video would provide more visual detail then most people are willing to put into on-screen text.  Of course the same is true for audio description - most people probably don't put enough into that either and would not even if they went the audio description route - that is because you only have to provide details that will fit in the pauses - at least some people read SC 1.2.5 to not require anything beyond what fits in the pauses - otherwise it would be an extended audio description which is required at Level AAA.

What I previously thought you were saying in your comments is that when video was present requiring description, that you could not use the media alternative approach.

I do like the option to use a media alternative route for meeting SC 1.2.5 as there are situations where a audio description would not be as useful or help and a written page as the primary document would be better.  For example, detailed instructions on writing computer code.   Videos are not good for that because of punctuation, etc. which is not spoken or would be clunky to speak.

Of course all of this does then raise the question of -- doesn't this just put us back in the situation where we are providing a full transcript and lieu of providing any audio description - we just have to call the full transcript the primary document and audio description can be put aside.

**Comment by alastc** [comment]
This didn't get traction for a normative update, so switching the labelling to the understanding doc.

**Comment by fstrr** [comment]
@bruce-usab @alastc What's happening with this issue? There's a [stale branch, last updated in June 2019](https://github.com/w3c/wcag/commit/791c1eb3ba6886b5e2b0ac28ec09104db028a6f3), that relates to the issue.

**Comment by kfranqueiro** [comment]
> What's happening with this issue? There's a [stale branch, last updated in June 2019](https://github.com/w3c/wcag/commit/791c1eb3ba6886b5e2b0ac28ec09104db028a6f3), that relates to the issue.

That branch was for PR #806, which didn't gain traction as referenced in Alastair's last comment, and has been closed/deleted.
